src/extensionParameterValidator.ts

```diff
@@ -59,16 +62,32 @@ function validateStringParameter(definition: StringParameter, value: string) {
     return SUCCESS_RESULT
 }
 
+function validateSelectParameter(definition: SelectParameter, value: string) {
+    const possibleValues = definition.options.map(option => option.id)
+    if (possibleValues.length === 0) {
+        return validationError("No option available for this parameter.")
+    }
+    if (possibleValues.includes(value)) {
+        return SUCCESS_RESULT
+    }
+    return validationError(`The value is not allowed. Possible values are ${possibleValues.join(', ')}`)
```

Comment:
If in typescript we can use a map function I would propose 

```suggestion
    const quotedValues = possibleValues.map(value => "'${value}'").join(', ')
    return validationError(`The value is not allowed. Possible values are ${quotedValues}.`)
```